ports/toziuha/gameinfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Toziuha Night - Dracula's Revenge.sh</path>
+    <name>Toziuha Night: Dracula's Revenge</name>
+    <desc>Toziuha Night Dracula's Revenge is a 2D side-scrolling action platformer with classicvania style. Travel through different linear maps set in a dark fantasy world; such as a gloomy forest and  Count Dracula's castle.
+Play as Xandria, a beautiful and skilled alchemist who, using an iron whip, fights against the most fearsome demons and other alchemists. Find out more about this tragic story of a girl who fights with a whip of iron chains, a cruel reminder of a tragic past.</desc>
+    <releasedate>20220321T000000</releasedate>
+    <developer> Danny Garay	</developer>
+    <publisher> Danny Garay	</publisher>
```

Comment:
The whitespaces can be removed I guess ?

```suggestion
    <developer>Danny Garay</developer>
    <publisher>Danny Garay</publisher>
```

Reply:
I guess they can